cargo-dist/templates/installer/installer.ps1.j2

```diff
@@ -254,6 +254,16 @@ function Invoke-Installer($bin_paths, $platforms) {
     Remove-Item "$bin_path" -Recurse -Force
     Write-Information "  $installed_file"
   }
+  {%- for source, dests in aliases | items %}
+  {%- for dest in dests %}
+
+  $source = Join-Path $dest_dir "{{ source }}.exe"
+  $dest = Join-Path $dest_dir "{{ dest }}.exe"
```

Comment:
That's fair, yeah. Not too much work to change it there for the Windows installer path.

Reply:
Oh yes, I did move this into Rust as requested. This involves a map of target triple to aliases, which we look up depending on what we install. It should make it possible for us to do the right thing in installers that can install for both Unix and Windows, like npm.

I considered putting it straight in executable zip fragments, rather than in one big map on InstallerInfo, but the way we mutate those in-place to alter their target triple made me want to avoid tacking any new target-specific stuff in there.